charts/internal/machine-controller-manager/seed/templates/deployment.yaml

```diff
@@ -83,6 +83,32 @@ spec:
         - mountPath: /var/lib/machine-controller-manager
           name: machine-controller-manager
           readOnly: true
+      - name: provider-openstack
+        image: {{ index .Values.images "machine-controller-manager-provider-openstack" }}
+        imagePullPolicy: Always
+        command:
+          - ./machine-controller
+          - --control-kubeconfig=inClusterConfig
+          - --namespace={{ .Release.Namespace }}
+          - --port={{ .Values.metricsPortOpenstack}}
+          - --target-kubeconfig=/var/lib/machine-controller-manager/kubeconfig
+          - --v=3
```

Comment:
Please add the following timeout flags explicitly [here](https://github.com/gardener/gardener-extension-provider-aws/pull/212/files#diff-e38adde4c814c39f0c3443b395a76dfc43639b003e8514c731b39e2b1683aea9R48-R50)